### PR TITLE
feat: Add calculator for DAO rewards

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/ckb-explorer-public-issues/issues/462

pc:

https://github.com/Magickbase/ckb-explorer-frontend/assets/11756869/9727a40f-e273-42ed-b5dc-ae2d7391df62

mobile:

https://github.com/Magickbase/ckb-explorer-frontend/assets/11756869/cb37459c-70ba-4cd4-af15-898035af69f7

### CODE REVIEW

**Comment by silySuper** [comment]
1.Actual page is different from UI design
Actual page:
截屏2024-03-28 16 47 22

UI design:
截屏2024-03-28 16 49 19

2.Address page transaction number shows 0,but actual has transactions

https://github.com/Magickbase/ckb-explorer-frontend/assets/25971273/e05fc7f1-ee18-4c11-a5ff-9260e3358c58

截屏2024-03-28 16 51 20

**Comment by devchenyan** [comment]
> 1.Actual page is different from UI design Actual page: 截屏2024-03-28 16 47 22
> 
> UI design: 截屏2024-03-28 16 49 19
> 
> 2.Address page transaction number shows 0,but actual has transactions @Daryl-L
> 
>  2024-03-28.16.52.46.mov 
> 截屏2024-03-28 16 51 20

1. There shouldn't be a slider here because the browser, unlike in neuron, has a limit of values that the user can deposit at most. Please update the design @Sven-TBD 

2. This should be fixed on the rgb++ branch @Daryl-L

**Comment by silySuper** [comment]
1.OK，In Nervos DAO Reward Calculator page, I find that it can not input decimals  when I click . in keyboard.

File: src/pages/NervosDao/DaoBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <div className={styles.actions}>
+          <button type="button" className={styles.btn} onClick={() => setShowRewardCalcutorModal(true)}>
+            {t('nervos_dao.reward_calculator')}
+            <GoIcon className={styles.icon} />
+          </button>
+          <button type="button" className={styles.btn} onClick={() => window.open(NERVOS_DAO_RFC_URL)}>
+            {t('nervos_dao.nervos_dao_rfc')}
+            <GoIcon className={styles.icon} />
+          </button>
+          <button type="button" className={styles.btn} onClick={() => window.open('https://dao.ckb.community/')}>
```
Comment (PainterPuppets) [inline src/pages/NervosDao/DaoBanner/index.tsx:26]: Maybe need to confirm if this link is right, `fund dao` is supposed to be `decentralized autonomous organization`, I see that here it should refer to the contract of `nervos dao`

**Comment by Keith-CY** [comment]
image
Deposit 1000 CKB but withdraw 1.7 CKB is misleading, expected to be 1001.7 CKB

**Comment by Keith-CY** [comment]
I've made the following updates

1. Use `compensation` instead of `reward` because `compensation` is generally used in Nervos DAO as rewards, e.g. the term `APC` is short for `Annual Percentage Compensation`, it avoids the attribute of finance.

2. The duration of charts is configurable
image

In the design, the duration is fixed to 1 month, which has been displayed above
image
It's unnecessary to repeat it in the chart again.

**Comment by Keith-CY** [comment]
This function has been reviewed by some users and got positive feedback. Will be pushed to testnet first for more feedback.

**Comment by Keith-CY** [comment]
Feel free to open a new issue if any problems should be fixed from your perspective @silySuper 
I merged it because it looks good to me enough for now.

**Pull request merged**